scripts/make_minized_sbc_base.sh

```diff
@@ -73,30 +73,10 @@ NO_BIT_OPTION='yes'
 
 source ${MAIN_SCRIPT_FOLDER}/common.sh
 
-verify_repositories
-verify_environment
-check_git_tag
+setup_project
 
-build_hw_platform
-create_petalinux_project
-configure_petalinux_project
-
-BOOT_METHOD='INITRD'
-BOOT_SUFFIX='_MINIMAL'
-INITRAMFS_IMAGE="avnet-image-minimal"
-configure_boot_method
-build_bsp
-
-BOOT_METHOD='INITRD'
-BOOT_SUFFIX='_FULL'
-INITRAMFS_IMAGE="avnet-image-full"
-configure_boot_method
-build_bsp
-
-BOOT_METHOD='EXT4'
-unset BOOT_SUFFIX
-unset INITRAMFS_IMAGE
-configure_boot_method
-build_bsp
+build_bsp 'INITRD' '_MINIMAL' "avnet-image-minimal"
```

Comment:
It's nice but I don't know if it's easier for customers to understand it (they have to look at the function in common.sh to understand the parameters).

Maybe we can come up with a more explicit way.

Reply:
Why the single quotes around 'INITRD' and '_MINIMAL' but double quotes around 'avnet-image-minimal'?

Reply:
That's just how they've always been. I can change them all to single for consistency